src/utils.rs

```diff
@@ -76,3 +76,26 @@ pub fn find_workspace(manifest: &Path, name: &str) -> Result<Option<PathBuf>, Er
     }
     Ok(None)
 }
+
+/**
+ * Search for .cargo/config.toml file
+ * 
+ * - Receive the workspace root path
+ */
+pub fn find_cargo_config(path: &Path) -> Result<Option<PathBuf>, Error> {
+    let path = dunce::canonicalize(path)?;
+    for config_folder_path in path
+        .ancestors()
+        .map(|dir| dir.join(".cargo"))
+        .filter(|dir| dir.exists())
+    {
+        for config_file_path in config_folder_path
+            .ancestors()
+            .map(|dir| dir.join("config.toml"))
+            .filter(|dir| dir.exists())
```

Comment:
Can this simply be collapsed to `.join(".cargo/config.toml")` in a single `for` loop?

I prefer using `.is_file()` over `.exists()`. Not sure if anyone else shares that opinion?

Reply:
`.exists ()` was already in another function. I just keep the pattern. 

Certainly the code can be improved, I just throw the first stone. I'm a beginner in rust, so I don't know much about its peculiarities yet.

Reply:
No worries, that's what review is for. I personally find it one of the best ways to teach others or learn more about a language, which is why I suggest even the most minute details :)